app/views/notifications/index.html.erb

```diff
@@ -68,6 +68,9 @@
     <%= render "articles/fitvids" %>
   </main>
   <%= javascript_packs_with_chunks_tag "followButtons", defer: true %>
+  <script>
+    <%== RunkitTag.script %>
```

Comment:
I'm guessing this can't be a deferred script and needs to be inlined like you have?

Reply:
TBH I've added it like this just because it's how it's added everywhere else it's used (article page, comment index page, article form).  I reckon if we can rework it, we should probably do it in a separate PR and look at it holistically